front-end/sass/_colors.sass

```diff
@@ -14,6 +14,23 @@ $color-success: #8ac159
 $color-success-dark: darken($color-success, 5%)
 $color-success-light: lighten($color-success, 5%)
 
+$color-status-info: #5087b9
```

Comment:
These variables seem very specific for the color stages template, can you move them to there?

Reply:
Since the colors are pretty much the same color as the originals, why not use the `darken(` function instead?

Reply:
only 5% is too less for the difference between the colors, but i will put them on the color-stages file